src/DDTrace/Transport/Stdout.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+namespace DDTrace\Transport;
+
+use DDTrace\Encoder;
+use DDTrace\Transport;
+
+final class Stdout implements Transport
```

Comment:
I like to have those kind of implementations for debugging purpose. It would have been very useful when I made my Laravel integration.

In my case I have some CLI scripts that generate traces and it will be very difficult to read debug traces with the standard output. I wonder if we can make a `Resource` transport that can accept Stdout / Stderr or any other resources.

Does it make sense?

Reply:
Ah, I follow, and yes, that makes sense. I'm going to make all of the output valid json so that you can slice it or pretty print it with jq or standard tools. I think `Stream` makes sense then as the name (http://php.net/manual/en/book.stream.php).